src/main/java/org/primeframework/mvc/security/cookiesession/SerializedSessionContainer.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2024, Inversoft Inc., All Rights Reserved
+ */
+package org.primeframework.mvc.security.cookiesession;
+
+import java.time.ZonedDateTime;
+import java.util.UUID;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+public class SerializedSessionContainer {
```

Comment:
`javadoc` and `@author` for consistency with this library.

Reply:
Done